CONTRIBUTING.md

```diff
@@ -77,4 +77,40 @@ When using the `cognite-toolkit` you have to build the configurations first and
     cdf-tk deploy --drop-data --drop --env=dev
     ```
 7. Verify in the UI (https://cog-power-ops.fusion.cognite.com/power-ops-staging/?cluster=bluefield.cognitedata.com&env=bluefield -> Data Models) that the changes are as expected.
-8. Follow the steps from point 7 on from above, where you post in #powerops-backend channel about the applied changes, execute `pygen_generate_clients.py`, etc.
+8. Follow the steps from point 7 on from above, where you post in #powerops-backend channel about the applied changes, execute `pygen_generate_clients.py`, etc.
+
+### Deploying to other environments
+
+Refer to the above sections for more context, PERFORM WITH CAUTION if you haven't done it before ask for support from other team members as these actions will delete many existing resources.
+
+Deploying to power-ops-dev, uses config `config.dev.yaml` and ensure `.env` credentials point to correct project
+
+   ```bash
+   cdf-tk build cognite/powerops --env dev --clean
+   cdf-tk deploy --drop-data --drop --env=dev --dry-run
+   cdf-tk deploy --drop-data --drop --env=dev

```

Comment:
These exact lines are already stated above.
What about not writing these full lines here and the ones below, but updating commands from lines 67, 71, and 77 with something like `cdf-tk build cognite/powerops --env <your environment> --clean`, and then listing all possible environments like staging, dev, heco-dev, and lyse-dev below, and keep saying which file will be used for which environment?

Reply:
These contributing guidelines I've added now are _only_ for the period in which we'll be deploying to customer environments manually so this is not the long term and will NOT be how it's done later. So I will leave it as is because the new section I added will be deleted once deployment pipelines are set up.

Reply:
Okay then. Go ahead and merge.

Reply:
One thing came into my mind now. Even though all these contributing guidelines will be removed later, let's keep some guidelines on what to consider when trying out changes to the data models by not messing up already existing populated data models, for example dev versus staging environment and modifying the space name, etc.